### DISCUSSION TITLE
blog/introducing-decap-cms-your-go-to-solution-for-content-management/

### DISCUSSION THREAD

**Discussion by giscus** [majesticooss/astros#18] [Blog]
# blog/introducing-decap-cms-your-go-to-solution-for-content-management/

Astros is a starter website template for Astro built with TailwindCSS and alpine js.

http://173.232.195.175/blog/introducing-decap-cms-your-go-to-solution-for-content-management/

**Comment by mkantowski** [comment]
hello

**Comment by mkantowski** [comment]
## test
